docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -1,4 +1,28 @@
-## I-1. Behind scene 1: Alice logs into Acme's Gameroom UI
+
+# Gameroom Technical Walkthrough
+
+***NOTE:*** This document is a work-in-progress! We are updating the
```

Comment:
Ungrammatical: No hyphens should be used in "work in progress"

Reply:
fixed